<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>patchSimpleDialog</class>
 <widget class="QDialog" name="patchSimpleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Falicsatlakozó, vagy patch panel felvitele</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,1">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="labelName">
         <property name="text">
          <string>Név :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelNote">
         <property name="text">
          <string>Megjegyzés :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditNote"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelZone">
         <property name="text">
          <string>Zóna :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBoxZone"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelPlace">
         <property name="text">
          <string>Helyiség : </string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineEditPlace">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxPlace"/>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonRefreshPlace">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="lv2g.qrc">
             <normaloff>:/icons/refresh.ico</normaloff>:/icons/refresh.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonNewPlace">
           <property name="text">
            <string>+</string>
           </property>
           <property name="icon">
            <iconset resource="lv2g.qrc">
             <normaloff>:/icons/add.ico</normaloff>:/icons/add.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButtonInfoPlace">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="lv2g.qrc">
             <normaloff>:/icons/edit.ico</normaloff>:/icons/edit.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelPlacePattern">
         <property name="text">
          <string>Helyiség minta :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEditPlacePattern"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelID">
         <property name="text">
          <string>ID :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLineEdit" name="lineEditID">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Index : </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="spinBoxFrom">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>-tól</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxTo">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>-ig</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Név minta :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="lineEditNamePattern">
           <property name="text">
            <string>J</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Offs.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxNameOffs">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Cimke minta :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="lineEditTagPattern"/>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Offs.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBoxTagOffs">
           <property name="minimum">
            <number>-1000</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAddPorts">
           <property name="text">
            <string>Port(ok) hozzáadása</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidgetPorts">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Név</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cimke</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Megosztás</string>
      </property>
      <property name="toolTip">
       <string>A hátlapi megosztás típusa</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Megosztva</string>
      </property>
      <property name="toolTip">
       <string>Hátlapi megossztás esetén az elsödleges port indexe</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Megjegyzés</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButtonDelPort">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Port(ok) törlése</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonShAB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>A-B megosztások</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonShDel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Megosztások törlése</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonShNC">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>NC nincs bekötve</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPort1">
       <property name="text">
        <string>Egy portos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPort2">
       <property name="text">
        <string>Két portos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonPort2Shared">
       <property name="text">
        <string>Két megosztott portos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="C" country="AnyCountry"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lv2g.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>patchSimpleDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>patchSimpleDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
